bats/tests/helpers/load.bash

```diff
@@ -20,10 +20,10 @@ source "$PATH_BATS_ROOT/bats-support/load.bash"
 source "$PATH_BATS_ROOT/bats-assert/load.bash"
 source "$PATH_BATS_ROOT/bats-file/load.bash"
 
+source "$PATH_BATS_HELPERS/os.bash"
+source "$PATH_BATS_HELPERS/utils.bash"
 # "defaults.bash" *must* be sourced before the rest of the files
```

Comment:
This comment looks more and more obsolete…

Reply:
I've added more fine-grained comments that should describe the dependencies more accurately. Hope this works better for you!